### PR TITLE
LAB-721 Updated Readme-s and screenshots for v2

### PR DESCRIPTION
**Pull request by pkuttappan**

### CODE REVIEW

File: docs/panel.png
Comment (jdevost) [inline docs/panel.png:1]: Can you redo this one, there's a typo (`Scra"pp"er`) in the header.

File: docs/copy_clipboard.png
Comment (jdevost) [inline docs/copy_clipboard.png:1]: Can you redo this one, there's a typo (`Scra"pp"er`) in the header.

File: misc/screenshots/sports.gif
Comment (jdevost) [inline misc/screenshots/sports.gif:1]: There's a typo (`Scra"pp"er`) in the header.

**Review by jdevost — COMMENTED** [review]
Looks good. Unfortunately, a typo made it in the screenshots.

**Comment by jdevost** [comment]
Also, can you update the screenshots under `misc/screenshots`? 
Those are displayed in the Chrome Web Store (https://chromewebstore.google.com/detail/web-scraper-helper-for-co/demelhhhnkocnphihjbhpafjnknknkdf?hl=en). 
Their requirements are
```
Up to a maximum of 5 
1280x800 or 640x400
JPEG or 24-bit PNG (no alpha)
At least one is required ```

**Comment by pkuttappan** [comment]
Good catch on the typo 😅 . Thanks. I have updated the screenshots and added new ones for the webstore.

**Review by jdevost — APPROVED** [review]

**Review by jdevost — APPROVED** [review]

**Pull request merged**